docs/university/course-four-workshop.md

```diff
@@ -0,0 +1,349 @@
+
+![Indicator Block Query Settings](/images/university/course-four-workshop/indicator-query.png)
+
+Once done, in the preview to the right, when the valve is reporting as open, we can see this:
+
+![Indicator Block Preview Good](/images/university/course-four-workshop/indicator-preview.png)
+
+**Exercise:**
+If the valve is closed, let’s do the exact opposite. Create the conditional to cause the Indicator Block to be red and display “closed” when the status is `false`.
+
+![Indicator Block Preview Bad](/images/university/course-four-workshop/indicator-preview-bad.png)
+
+After completing the exercise, add the block to the dashboard.
+
+### Complete
+
+Now, our dashboard is complete! If all went well, we should have something that looks like this:
+
+![Complete Dashboard](/images/university/course-four-workshop/dashboard-complete-one.png)
+
+## Part Two: Context Variables
+
+Currently we only have one device and one dashboard. But, what if we have multiple devices? As you may recall from the videos, we don’t want to create a dashboard for each device, we can use [context variables](/dashboards/context-variables/) to make this dashboard reusable.
+
+### More Devices
+
+Now, we are going to create a reusable dashboard. But first, we need a new device. If you go to your current device settings, you can clone your device.
+
+![Clone Device](/images/university/course-four-workshop/clone-device.png)
+
+Also, to make our devices more easily identifiable, let’s rename the devices to “Water Pump 0” and “Water Pump 1”.
+
+![Rename Water Pump](/images/university/course-four-workshop/water-pump-rename.png)
+
+Before we change our dashboard, our new device needs to be reporting data to Losant. To do so, we can use the simulator.
+
+### Simulate Extra Device Exercise:
+
+Your task is simple: simulate data for “Water Pump 1” (the newly created device).
+
+**Remember, the simulator is using your browser window as the device. So, you must leave this browser window open for your device to simulate data. This means you should have two independent tabs at this point generating data.**
+
+Next, we can add a Device List Block to the dashboard to display a list of all of our devices.
+
+### Add a Device list Block
+
+On the “Add Block” page, choose the Device List Block:
+
+![Device List Block Add](/images/university/course-four-workshop/device-list.png)
+
+In the block settings, we can optionally choose to filter our devices, or if we don’t filter, the list will include all of our devices. Since we only have two devices, we can leave this blank.
+
+The items in the list can be a link. We will change this later, but for now, just leave it as a link to the device page.
+
+![Device List Block Settings](/images/university/course-four-workshop/device-list-settings.png)
+
+Here is what your preview should look like:
```

Comment:
I could (should?) have called this earlier but you switch between second-person singular and first-person plural tenses a lot here. This line, and the one that follows, is a good example.

Reply:
Disregard.